modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -1028,29 +1027,37 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
     @Story("Account transfers domain ownership")
     @SdkTestId("transfer_domain_ownership")
     fun `transfer domain ownership`(): Unit = runBlocking {
-        val genesisAccountId = AccountId(GENESIS.asDomainId(), GENESIS.asName())
-        var domain = QueryBuilder.findDomainById(DEFAULT_DOMAIN_ID)
+        val domainId = "Kingdom".asDomainId()
```

Comment:
let's better lift the literal to a const (like `DEFAULT_DOMAIN_ID`)

Reply:
why? it's just a value local to this test

Reply:
I see, thought it might be used in some other tests but seems not